### PR TITLE
update docs for static analysis data schema

### PR DESCRIPTION
**Pull request by maxfisher-g**

### CODE REVIEW

File: docs/data_schema.md [hunk 1]
```diff
@@ -143,16 +145,13 @@ The top-level identifying keys have been revised a little bit from the dynamic a
...
-          "description": string,
-          "size": int,
-          "sha256": string,
-          "line_lengths": [
-            { "value": int, "count": int }
-          ]
-        },
-        "parsing": {
-          "language": string,
+	    "detected_type": string,
```
Comment (calebbrown) [inline docs/data_schema.md:148]: nit: the indents are a bit inconsistent.

File: docs/data_schema.md [hunk 2]
```diff
@@ -220,41 +217,63 @@ Contains all result data from the static analysis; see description below
 ### `results` object
 
 #### `files`
-List of static analysis results, one per file contained in the analyzed package tarball. Files are enumerated in lexical order. Symlinks or special files such as device files, sockets and pipes are excluded. Each item is corresponds to a SingleResult object in Go; see description below.
+List of static analysis results, one per file contained in the analyzed package tarball. Files are enumerated in lexical order. Symlinks or special files such as device files, sockets and pipes are excluded. Each item is corresponds to a FileResult object in Go; see description below.
```
Comment (calebbrown) [inline docs/data_schema.md:220]: nit: `Each item is corresponds` -> `Each item corresponds`

File: docs/data_schema.md [hunk 3]
```diff
@@ -220,41 +217,63 @@ Contains all result data from the static analysis; see description below
...
-### `SingleResult` object
+### `FileResult` object
 
 #### `filename`
 Path of the file, relative to the package archive root
 
-#### `basic` (optional)
-Contains results from the basic static analysis task; this is information that can be obtained without parsing the file. See basic object description below. Field is omitted if the basic analysis task was excluded during the analysis run.
+#### `detected_type`
+Output of `file` command run on the file. Omitted if the `basic` analysis task was not run.
```
Comment (calebbrown) [inline docs/data_schema.md:228]: Might be worth mentioning that it is the file command included by Ubuntu.

**Review by calebbrown — APPROVED** [review]

File: docs/data_schema.md (hunk 1 above)
Comment (maxfisher-g) [reply docs/data_schema.md:148]: fixed, had accidentally inserted tabs

File: docs/data_schema.md (hunk 2 above)
Comment (maxfisher-g) [reply docs/data_schema.md:220]: done, thanks

File: docs/data_schema.md (hunk 3 above)
Comment (maxfisher-g) [reply docs/data_schema.md:228]: done

**Pull request merged**